models/public/wav2vec2-base/README.md

```diff
@@ -0,0 +1,100 @@
+# wav2vec2-base
+
+## Use Case and High-Level Description
+
+Wav2Vec2.0-base is a model, which pre-trained to learn speech representations on unlabeled data as described in [wav2vec 2.0: A Framework for Self-Supervised Learning of Speech Representations](https://arxiv.org/abs/2006.11477) paper and fine-tuned for speech recognition task with a Connectionist Temporal Classification (CTC) loss on LibriSpeech dataset containing 960 hours of audio.
+The model is composed of a multi-layer convolutional feature encoder which takes as input raw audio and outputs latent speech representations, then fed to a Transformer to build representations capturing information from the entire sequence. For base model Transformer consists of 12 transformer layers and has 768 as feature dimension.
+For details please also check [repository](https://github.com/pytorch/fairseq/tree/master/examples/wav2vec#wav2vec-20) and [model card](https://huggingface.co/facebook/wav2vec2-base-960h).
+
+## Specification
+
+| Metric           | Value              |
+| ---------------- | ------------------ |
+| Type             | Speech recognition |
+| GFLOPs           | 26.843             |
+| MParams          | 94.3965            |
+| Source framework | PyTorch\*          |
+
+## Accuracy
+
+| Metric                       | Value |
+| ---------------------------- | ----- |
+| WER @ Librispeech test-clean | 3.39% |
+
+### Input
+
+#### Original model
+
+Normalized audio signal, name - `inputs`,  shape - `1, 30480`, format is `B, N`, where:
+
+- `B` - batch size
```

Comment:
Most models used `N` for batch size. Is there any explanation why it `B` here?

Reply:
because I use N for the second parameter, it is hard to explain number of samples of sequence length with another latter. If you have better suggestion, I will be happy

Reply:
BTW, my description format is aligned with another speech model - quartznet https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/public/quartznet-15x5-en

Reply:
May be you could use the `N, L`, like in [aclnet](https://github.com/openvinotoolkit/open_model_zoo/blob/develop/models/public/aclnet/README.md) model?